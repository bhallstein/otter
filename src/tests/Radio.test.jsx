import React from 'react';
import Radio from '../generic/Radio';
import TestRenderer from 'react-test-renderer';

it('renders correctly', () => {
  // mocks
  const values = {
    content: '',
  };
  const data = {
    label: '',
    options: [],
  };
  const onChange = () => {
    return;
  }

  const tree = TestRenderer
    .create(<Radio action={onChange} data={data} values={values}></Radio>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});