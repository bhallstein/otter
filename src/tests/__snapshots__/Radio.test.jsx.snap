// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="di mt3 mr3 tl"
>
  <label>
    
  </label>
  <div
    className="di f3 ml3"
  />
</div>
`;
